### PR TITLE
feat: implement fulfillment user navigation item

### PR DESCRIPTION
**Pull request by aleespryker**
Closes [HRZ-2896](https://spryker.atlassian.net/browse/HRZ-2896)

[HRZ-2896]: https://spryker.atlassian.net/browse/HRZ-2896?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: libs/base/ui/overlays/modal/src/modal.styles.ts [hunk 1]
```diff
@@ -41,6 +41,10 @@ export const styles = css`
   header {
     align-items: center;
     justify-content: space-between;
+    flex-direction: var(--oryx-header-direction);
+    text-align: var(--oryx-header-text-align);
+    font-size: var(--oryx-header-font-size);
+    font-weight: var(--oryx-header-font-weight);
```
Comment (Sinucid) [inline libs/base/ui/overlays/modal/src/modal.styles.ts]: Need to align with Nuwan and justify designs for modal component

File: libs/domain/picking/src/components/picking-header/picking-header.component.ts [hunk 2]
```diff
@@ -48,14 +48,7 @@ export class PickingHeaderComponent extends PickingListMixin(LitElement) {
...
       ${this.renderCartNoteButton()}
-      <oryx-site-navigation-item
-        uid="user-profile"
-        .options=${{
-          icon: IconTypes.Profile,
-          triggerType: 'icon',
-          contentBehavior: 'modal',
-        }}
-      ></oryx-site-navigation-item>`;
+      <oryx-picking-user-navigation-item> </oryx-picking-user-navigation-item>`;
```
Comment (Sinucid) [inline libs/domain/picking/src/components/picking-header/picking-header.component.ts]: use ordinary `oryx-site-navigation-item`

File: libs/domain/picking/src/components/user-navigation-item/user-navigation-item.component.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  static styles = styles;
+
+  protected override render(): TemplateResult {
+    return html`<oryx-site-navigation-item
+      uid="user-profile"
+      .options=${{
+        icon: IconTypes.Profile,
+        triggerType: 'icon',
+        contentBehavior: 'modal',
+        fullscreen: true,
```
Comment (Sinucid) [inline libs/domain/picking/src/components/user-navigation-item/user-navigation-item.component.ts]: not needed. Will be the standard behaviour of modal component for small screens

File: libs/domain/picking/src/components/user-navigation-item/user-navigation-item.component.ts [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+        heading: i18n('oryx.picking.account'),
```
Comment (Sinucid) [inline libs/domain/picking/src/components/user-navigation-item/user-navigation-item.component.ts]: use label instead and update navigation item component to make it use label as heading for the modal

File: libs/domain/site/navigation-item/src/navigation-item.styles.ts [hunk 5]
```diff
@@ -20,4 +20,8 @@ export const styles = css`
   oryx-dropdown {
     --oryx-popover-maxwidth: 188px;
   }
+
+  oryx-modal {
+    --oryx-icon-color: var(--oryx-color-neutral-11);
```
Comment (Sinucid) [inline libs/domain/site/navigation-item/src/navigation-item.styles.ts]: not needed. Will be updated with adding the semantic types for icon buttons

**Review by Sinucid — CHANGES_REQUESTED** [review]

**Review by gund — DISMISSED** [review]

File: libs/domain/picking/src/components/user-navigation-item/user-navigation-item.component.ts (hunk 3 above)
Comment (Sinucid) [reply libs/domain/picking/src/components/user-navigation-item/user-navigation-item.component.ts]: You can just hardcode fullscreen attr on oryx-modal inside navigation-item for now

**Review by Sinucid — APPROVED** [review]

**Pull request merged**